docs/openapi.yml

```
@@ -570,7 +570,7 @@ paths:
                 reference:
                   type: string
             example:
-              phone_number: "2028675309"
+              phone_number: "202-867-5309"
```

😆 we still have a [Tommy Tutone reference](https://en.wikipedia.org/wiki/867-5309/Jenny)...

Let's change this to truly be fake:

```suggestion
              phone_number: "800-555-0100"
```